src/docs/sdk/event-payloads/stacktrace.mdx

```diff
@@ -24,6 +24,12 @@ follow this rule of thumb:
 : **Required**. A non-empty list of stack frames (see below). The list is
   ordered from caller to callee, or oldest to youngest. The last frame is the
   one creating the exception.
+  
+  <Note>
+    <markdown>
```

Comment:
I think these tags / React components do not exist in the "develop" repository, that's why it doesn't produce the desired effect.

Reply:
Ah, that makes sense. Thanks. 😄